### DISCUSSION TITLE
[@types/aws-lambda] - lexv2 - LexV2Result definition is missing slots

### DISCUSSION THREAD

**Discussion by arnaldo82** [DefinitelyTyped/DefinitelyTyped#58076] [Issues with a @types package]
Hi,
According to [amazon documentation](https://docs.aws.amazon.com/lexv2/latest/dg/lambda.html#lambda-response-format)
the lambda response can propagate the slots already fulfilled by the client.
This is useful to keep the status of slots among the session and simplify the validation logic. 
 
Add slots array to LexV2Result in types/aws-lambda/trigger/lex-v2.d.ts

Example:

```
{
    sessionState: {
      sessionAttributes,
      dialogAction: {
        type: 'Delegate',
      },
      intent: {
        name: intentName, 
        state: 'ReadyForFulfillment',
        slots: mySlotsUntilNow
      }  
    }
  .....
  }
```

Thanks and best regards,
Arnaldo

[a comment by an automated account is not shown]

**Comment by jamesorlakin** [comment]
This should now be resolved by the merging of #58168
